dev/com.ibm.ws.jsfContainer_fat_2.3/publish/files/MANIFEST_badMyfacesApi_40.MF

```diff
@@ -0,0 +1,93 @@
+Manifest-Version: 1.0
+Archiver-Version: Plexus Archiver
+Created-By: Apache Maven Bundle Plugin
+Built-By: volosied
+Build-Jdk: 11.0.17
+Specification-Title: Apache MyFaces Core 4.0 - API
+Specification-Version: 3.0.0
```

Comment:
Test looks at this `Specification-Version` which is 3.0 here when it should be 4.0.

Test expects a failure.